src/Xamarin.Android.Build.Tasks/Tasks/RemoveDirFixed.cs

```diff
@@ -61,11 +60,15 @@ public override bool RunTask ()
 				} catch (DirectoryNotFoundException ex) {
 					// This could be a file inside the directory over MAX_PATH, we can attempt using the \\?\ syntax
 					if (OS.IsWindows) {
-						fullPath = Files.ToLongPath (fullPath);
-						Log.LogDebugMessage ("Trying long path: " + fullPath);
-						Directory.Delete (fullPath, true);
-						temporaryRemovedDirectories.Add (directory);
-						continue;
+						try {
+							fullPath = Files.ToLongPath (fullPath);
+							Log.LogDebugMessage ("Trying long path: " + fullPath);
+							Directory.Delete (fullPath, true);
+							temporaryRemovedDirectories.Add (directory);
+							continue;
+						} catch (Exception inner) {
+							Log.LogErrorFromException (inner);
+						}
 					}
 					Log.LogErrorFromException (ex);
```

Comment:
This will cause both errors to get logged, which may be desired?

Reply:
I think we want both logged. Is two errors fine, or should I use `AggregateException`?

All these calls should also be doing:
```csharp
Log.LogUnhandledException (TaskPrefix, ex);
````
I will change that.